offchain-modules/packages/x/src/multisig/multisig-mgr.ts

```diff
@@ -8,11 +8,13 @@ import { httpRequest } from './client';
 
 const SigErrorTxNotFound = 1003;
 const SigErrorTxUnconfirmed = 1004;
+const SigErrorBlockSyncUncompleted = 1005;
 const SigErrorCodeUnknownError = 9999;
 
 const retryErrorCode = new Map<number, boolean>([
```

Comment:
there is a `Set` data structure in JS, maybe we can just use a set for managing `RetryErrorCode`, or will the value corresponding to the error code be extended with functions, such as retryTimes, in which case it should be marked with a TODO  here?